src/components/directory-content.js

```diff
@@ -639,14 +647,14 @@ const DirectoryContent = () => {
         return [...new Set(acc)];
     };
 
-    const getCurrentChildrenWithNotClickableRows = () => {
-        return currentChildren.map((child) => {
-            return {
+    const rows = useMemo(
+        () =>
+            currentChildren?.map((child) => ({
```

Comment:
We could override the `child.type` value here in rows calling `getElementTypeTranslation` , instead of using a `typeCellRender` which is called too late after the sorting...
Then we avoid customizing the `sort` method at all...
We already assume that the data (`rows`) passed to the table is not strictly equal to `currentChildren` content (`notClickable`) then I think we could totally do this.